src/ProtoRoute/Message.hs

```diff
@@ -34,6 +34,9 @@ data Field = Field
 fromMN :: MsgName -> String
 fromMN (MN str) = str
 
+fromFN :: FieldName -> String
+fromFN (FN str) = str
```

Comment:
You can define your `newtype` with record field so you won't need to write getters explicitly. Like this:

```haskell
newtype FieldName = FN { unFieldName :: String } deriving ...
-- `un` is a shorter version of `unwrap`; this prefix is usually used by convention for newtypes
```